doc/user/content/connect-sources/cdc-postgres-direct.md

```diff
@@ -108,51 +160,100 @@ As a _superuser_ (`cloudsqlsuperuser`):
 
 {{< /tabs >}}
 
-Once logical replication is enabled:
+## Step 2. Create a publication
 
-1. Grant the required privileges to the replication user:
+Once logical replication is enabled, the next step is to create a publication with the tables that you want to replicate to Materialize. You'll also need create a user for Materialize and give the user permissions to manage replication.
+
+{{< tabs >}}
+{{< tab "AWS RDS">}}
+
+1. For each table that you want to replicate to Materialize, set the [replica identity](https://www.postgresql.org/docs/current/sql-altertable.html#SQL-ALTERTABLE-REPLICA-IDENTITY) to `FULL`:
 
     ```sql
-    ALTER ROLE "dbuser" WITH REPLICATION;
+    ALTER TABLE table1 REPLICA IDENTITY FULL;
     ```
 
-    And ensure that this user has the right permissions on the tables you want to replicate:
+    This setting makes sure sufficient information is written to the replication stream for  `UPDATE` and `DELETE` operations.
```

Comment:
Can you add some extra context here? I'm not sure what sufficient information means.

Reply:
I think a good thing to link to for users to understand this is https://www.postgresql.org/docs/current/logical-replication-publication.html, though I think it warrants context that even though PG makes this sound scary, it's actually very beneficial for Materialize because it gives PG sources more powerful semantics (e.g. being able to UPSERT) while not requiring nearly as much memory.